src/compiler.rs

```diff
@@ -118,7 +88,7 @@ pub fn compile_sync(
   let res_msg = recv_future.wait().unwrap().unwrap();
 
   let res_json = std::str::from_utf8(&res_msg).unwrap();
-  CodeFetchOutput::from_json(res_json)
```

Comment:
What do you think about renaming the CodeFetchOutput on the Rust side to ModuleMetaData too?

Reply:
I sort of changed it, but I see that isn't complete.